src/main/java/org/broadinstitute/dsm/jobs/TestBostonUPSTrackingJob.java

```diff
@@ -170,7 +177,7 @@ private static void logError(String trackingId, UPSError[] errors) {
         logger.error(errorString);
     }
 
-    private static void updateStatus(@NonNull Connection conn, String trackingId, String oldType, UPSTrackingResponse response, boolean isReturn, DdpKit kit, DDPInstance ddpInstance) {
+    private static void updateStatus(String trackingId, String oldType, UPSTrackingResponse response, boolean isReturn, DdpKit kit, DDPInstance ddpInstance) {
```

Comment:
Can you re-introduce the `Connection` here or on L206?  This code might be re-usable with the connection, and internal to this method it doesn't look like there is any external IO.

Reply:
moved it `updateTrackingInfo` so that when we reuse it we don't need any ups objects